samples/hello-world/HW-pasqal-pulser.ipynb

```diff
@@ -0,0 +1,359 @@
+   ],
+   "id": "c3d23fc53155e243"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": "### 3. Submit the quantum program to Pasqal",
+   "id": "95f8578dfb2660f4"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Using the Pasqal simulator target, call \"run\" to submit the job. We'll\n",
+    "# use 10 repetitions (simulated runs).\n",
+    "input_args = dict(\n",
+    "   input_data=dict(sequence_builder=seq.to_abstract_repr()),\n",
+    "   input_params={\"shots\": 100}, # Number of shots\n",
+    ")\n",
+    "\n",
+    "job = target.submit(\n",
+    "    **input_args, \n",
+    "    input_data_format=\"pasqal.pulser.v1\", \n",
+    "    output_data_format=\"pasqal.pulser-results.v1\",\n",
+    "    name=\"Pasqal sequence\",\n",
+    ")\n",
+    "\n",
+    "# Print the job ID.\n",
+    "print(\"Job id:\", job.id)"
+   ],
+   "id": "ed2e79aac388baa9"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "The job ID can be used to retrieve the results later using the [get_job method](https://learn.microsoft.com/python/azure-quantum/azure.quantum.workspace?#azure-quantum-workspace-get-job) or by viewing it under the **Job management** section of the portal."
+   ],
+   "id": "bb4c8c91778444da"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "### 4. Obtain the job results\n",
+    "Let's await the job execution by calling `job.get_results()`. This may take a minute or so ⏳. Your job is being packaged and sent to Pasqal, where it will wait its turn to be run."
+   ],
+   "id": "f2111a90ca2d482a"
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Await job results.\n",
+    "print(\"Awaiting job results...\")\n",
+    "job.wait_until_completed()\n",
+    "result = job.get_results()\n",
+    "\n",
+    "# To view each measured qubit state, you can print the result.\n",
+    "print(\"Job finished. Measure count for each state:\")\n",
+    "print(result)"
+   ],
+   "id": "4bf1813d0e966f42"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": "**See the result above? Congratulations, you've submitted a Pasqal job with Azure Quantum! 👏**",
+   "id": "a13268e76598db7b"
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Estimate costs\n",
+    "\n",
+    "To estimate the costs of running this program on a simulator or hardware, you can use the `target.estimate_cost` method.\n",
+    "\n",
+    "Note that cost emulation for Pasqal emulators has not been implemented yet."
```

Comment:
Correct if I'm misunderstanding this, but if we don't support cost estimation for emulators and we add a cell to estimate cost for an emulator target `pasqal.sim.emu-tn` we use in this Notebook, user won't see results.
Might be we could remove this cell since the estimation won't work for this target.

Reply:
Indeed the output of that cell is:

>Price estimation is not implemented yet for this target.

So there is no "crash" per se, only an information message. My idea was to keep that code in the notebook so that the user learns about that azure-quantum sdk method, in the hope that they will remember to use it later, before running a job on a QPU.

As you prefer.

Reply:
I have applied your latest suggestion, showing cost estimates for fresnel.

Sadly, I receive the same message (not implemented) :pensive:

I think I need to open an issue on our side to implement at least that.

Reply:
Oh, if PASQAL doesn't yet have cost estimation implemented for any of the targets, might be we could remove this cell altogether for now. Like I mentioned above, we don't typically advertise cells that are work-in-progress

Reply:
Sure, I removed step 5 completely, and opened an issue on our side.